packages/angular/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+    "compilerOptions": {
+        "strict": true,
+        "target": "ES2015",
```

Comment:
This can be ES2018 since Node supports it.

Reply:
cool